src/sql/src/func.rs

```diff
@@ -2571,6 +2571,10 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
             params!(Float64) => UnaryFunc::TruncFloat64(func::TruncFloat64) => Float64, 1343;
             params!(Numeric) => UnaryFunc::TruncNumeric(func::TruncNumeric) => Numeric, 1710;
         },
+        "try_parse_monotonic_iso8601_timestamp" => Scalar {
```

Comment:
This should go under MZ_CATALOG_BUILTINS.

Reply:
done!